app/components/Dashboard/AssetBalancesPanel/AssetBalancesPanel.jsx

```diff
@@ -99,11 +107,35 @@ export default class AssetBalancesPanel extends React.Component<Props> {
     return neoPrice && NEO !== '0' ? neoPrice * toNumber(NEO) : 0
   }
 
+  getNEOFormattedPriceChange = (): string => {
+    return (this.props.neoPriceChange >= 0 ? '+' : '') + (this.props.neoPriceChange * 100).toFixed(2) + '%'
```

Comment:
Its a bit hard to read, how about:
`const { neoPriceChange } = this.props` 
and then using ES6 template for the contact?

Also, better use `bignumber.js` here.